app/pods/publications/index/template.hbs

```diff
@@ -33,6 +33,7 @@
   <div class="vlc-u-box-model-maximize-height vlc-scroll-wrapper">
     {{#if this.isShowPublicationModal}}
       <Publications::NewPublicationModal
+        @getPublicationNumber={{this.getNewPublicationNextNumber}}
```

Comment:
Ik zou de `getNewPublicationNextNumber` helper method gewoon naar de `Publications::NewPublicationModal` component verhuizen ipv als argument mee door te geven. Of wordt ze nog ergens gebruikt in deze template?

Reply:
ik dacht dat je vanuit een component niet aan de store kon. Maar dat blijkt dus wel te gaan :-)